trino/dbapi.py

```diff
@@ -584,66 +603,22 @@ def describe(self, sql: str) -> List[DescribeOutput]:
 
         return list(map(lambda x: DescribeOutput.from_row(x), result))
 
-    def genall(self):
-        return self._query.result
+    def genall(self) -> Any:
+        if self._query:
+            return self._query.result
+        return None
 
-    def fetchall(self) -> List[List[Any]]:
+    def fetchall(self) -> Optional[List[List[Any]]]:
         return list(self.genall())
 
-    def cancel(self):
+    def cancel(self) -> None:
         if self._query is None:
             raise trino.exceptions.OperationalError(
                 "Cancel query failed; no running query"
             )
         self._query.cancel()
 
-    def close(self):
+    def close(self) -> None:
         self.cancel()
         # TODO: Cancel not only the last query executed on this cursor
         #  but also any other outstanding queries executed through this cursor.
-
-
-Date = datetime.date
```

Comment:
Why removed? These are standard DB-API types and users can inspect these to understand what they'll receive and what they need to pass.

Reply:
removal reverted